sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_models.hpp

```diff
@@ -970,4 +970,29 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
     IssuerProperties Properties;
   };
 
+  /**
+   * @brief The contact information for the vault certificates.
+   *
+   */
+  struct CertificateContact
+  {
+    /**
+     * @brief Contact email address of.
+     *
+     * @remark Email is required, has to be unique
```

Comment:
```suggestion
     * @remark Email is required and has to be unique.
```

What happens when it isn't unique?